src/biogeophys/PhotosynthesisMod.F90

```diff
@@ -346,8 +346,8 @@ subroutine InitAllocate(this, bounds)
       ! statements.
       allocate(this%vcmx25_z_patch  (begp:endp,1:nlevcan)) ; this%vcmx25_z_patch    (:,:) = 30._r8
       allocate(this%jmx25_z_patch   (begp:endp,1:nlevcan)) ; this%jmx25_z_patch     (:,:) = 60._r8 
-      allocate(this%vcmx_prevyr     (begp:endp,1:nlevcan)) ; this%vcmx_prevyr       (:,:) = 30._r8
-      allocate(this%jmx_prevyr      (begp:endp,1:nlevcan)) ; this%jmx_prevyr        (:,:) = 60._r8
+      allocate(this%vcmx25_z_last_valid_patch     (begp:endp,1:nlevcan)) ; this%vcmx25_z_last_valid_patch       (:,:) = 30._r8
```

Comment:
- [ ] these variable names seem descriptive, as @billsacks was asking

Reply:
@wwieder are you asking for a change here, or approving this? Your comment makes it sound like you're approving this change, but you put a check box in....

Reply:
sorry, just responding to a question @negin513 had in the PR

Reply:
> * [x]  these variable names seem descriptive, as @billsacks was asking

@wwieder Perfect! Thanks for your comment.